tests/foreman/ui/test_rhcloud_inventory.py

```diff
@@ -25,6 +25,33 @@
 )
 
 
+@pytest.fixture(scope='module', autouse=True)
```

Comment:
@SatelliteQE/airgun-tier-2-reviewers Can one of you confirm my logic here. Basically I want to be able to pass in data_collection_minimal whenever I want to set that setting to Yes. After the tests finishes(or fails) I want to make sure that setting is set back to NO.

I also created a another fixture scoped at the module level. To make sure that setting is set to NO before other tests get run.

Reply:
As long as you are seeing proper behavior in the tests (the settings are how you expect) then this should be fine. Logic looks good to me.